gcc/rust/ast/rust-item.h

```diff
@@ -2006,6 +2006,8 @@ class EnumItem : public VisItem
 
   location_t locus;
 
+  std::unique_ptr<Expr> expression;
```

Comment:
If this expression is optional it should be handled as truly optional and not rely on null pointers to express a missing value, we already have a bunch of rogue null pointer in the codebase, I'd rather be sure some value can be missing than spend some time figuring out whether a null pointer is a bug or an intended behavior. Please wrap it into an `tl::optional`.